tests/test_ansible.py

```diff
@@ -5,7 +5,7 @@
 from tox_ansible.ansible import Ansible
 
 
-@pytest.mark.parametrize("scenarios,expected", [([1], True), ([], False)])
+@pytest.mark.parametrize("scenarios,expected", [([1], True), ([], True)])
```

Comment:
Do we want to add a test scenario now that tests for expected == False, since we're now running sanity tests everywhere that Ansible is detected?